### PR TITLE
[18.0][MIG] website_sale_checkout_skip_payment: Migration to 18.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT51372

@CarlosRoca13 @carlos-lopez-tecnativa please review

### CODE REVIEW

**Comment by pilarvargas-tecnativa** [comment]
I have to reduce administrative commits, I know.

**Comment by pedrobaeza** [comment]
And check tests

/ocabot migration website_sale_checkout_skip_payment

**Comment by pilarvargas-tecnativa** [comment]
@chienandalu can you take a look at it?

**Comment by CarlosRoca13** [comment]
:+1: Thanks

**Review by CarlosRoca13 — APPROVED** [review]

File: website_sale_checkout_skip_payment/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module allows to logged users to checkout with no payment step. At
+the end of the checkout proccess the quotation is sent to the user email
+address and set to *Qoutation Sent* state.
```
Comment (carlos-lopez-tecnativa) [inline website_sale_checkout_skip_payment/readme/DESCRIPTION.md:3]:
```suggestion
address and set to *Quotation Sent* state.
```

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM only review the ` DESCRIPTION.md`

**Comment by pilarvargas-tecnativa** [comment]
I think this commit is the culprit: https://github.com/OCA/e-commerce/pull/976/commits/4f11120868f48f60f12805a2847fe5fad08a346e

Although corrected in this previous: https://github.com/OCA/e-commerce/pull/976/commits/24b04439364e5512dd2a77e01ab92f1a27c9de2d

@pedrobaeza in this case I remove that commit from the bot?

**Comment by pedrobaeza** [comment]
Squash the commit history properly to remove noise.

File: website_sale_checkout_skip_payment/readme/DESCRIPTION.md (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply website_sale_checkout_skip_payment/readme/DESCRIPTION.md:3]: Done!

**Pull request merged**